postgres/manage_users.py

```diff
@@ -4,21 +4,27 @@
 from botocore.exceptions import ClientError
 
 
-def drop_user(cursor, username):
+def create_or_update_db_user(conn, cursor, username, password, permissions):
     cursor.execute(f"SELECT * FROM pg_catalog.pg_user WHERE usename = '{username}'")
```

Comment:
I think there is a misspelling here `WHERE usename =` it should be `WHERE username = ` right?

Reply:
The inbuilt pg_catalog.pg_user table actually has a column, `usename`!